### ISSUE TITLE
Missing file

### ISSUE THREAD

**Issue by int-0** [bitcraft/retrogamelib#5]
Bubbman example missing a file called "data/bubbman-4.png".

**Comment by bitcraft** [comment]
Yes.  This repo is a basic update of a very old project from another person.   I've double checked my repo, and the source, and I was not able to find that particular file.  Here is the original source: https://code.google.com/archive/p/retrogamelib/source/default/source
